snapshot.c

```diff
@@ -22,6 +23,137 @@ int rdbSave(const char *filename, void *info);
 
 /* ------------------------------------ Snapshot metadata ------------------------------------ */
 
+void initSnapshot(RedisRaftCtx *ctx)
```

Comment:
One thing I realized while reviewing is there's some confusion about inbound snapshot files and locally generated/outbound snapshot files. Maybe it will be easier to follow if function/var names would indicate that.

Reply:
Changed function names a bit.